app/controllers/api/v1/recent_jobs_controller.rb

```diff
@@ -1,13 +1,9 @@
 class Api::V1::RecentJobsController < ApplicationController
-
   def index
-    if params[:location]
-      search_location = params[:location]
-      jobs_by_location = Job.last_month.by_location(search_location)
-      render json: jobs_by_location
-    else
-      paginate json: Job.last_month, per_page: 25
-    end
-  end
+    jobs = Job.last_month
+    jobs = jobs.merge Job.by_location(params[:location]) if params[:location]
```

Comment:
I like this. That's a 3 for 1 type of action you got there. I'll merge it but if we push it up to master right now will it effect the rake task I have running now? @hhoopes I guess I could push it up and see if I get an error or something.